index.html

```diff
@@ -1,19 +1,122 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <meta
+      name="description"
+      content="Karma: A new way to boost your productivity with WiFi"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <header class="header-section">
+      <img
+        class="company-logo"
+        src="img/karma-logo.svg"
+        alt="Karma company logo"
+      />
+      <nav class="navbar">
+        <button id="nav-hamburger-Menu" class="navbar-hamburger-Menu">
+          <img
+            class="hamburger-icon"
+            src="img/menu-hamburger.svg"
+            alt="Hamburger icon"
+          />
+        </button>
+        <ul id="nav-full-Menu" class="navbar-full-Menu">
+          <li><a href="#">Meet Karma</a></li>
+          <li><a href="#">How it Works</a></li>
+          <li><a href="#">Store</a></li>
+          <li><a href="#">Blog</a></li>
+          <li><a href="#">Help</a></li>
+          <li><a href="#">Login</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="hero-section">
+        <h1>Introducing Karma</h1>
+        <p>Bring WiFi with you, everywhere you go.</p>
+        <button>Learn More</button>
+      </section>
+      <section class="about-section">
+        <h2>Everyone needs a little Karma.</h2>
+        <article class="about-articles">
+          <div class="about-container">
+            <img
+              class="about-icon"
+              src="img/icon-devices.svg"
+              alt="Picture of a smartphone, table and desktop computer"
```

Comment:
```suggestion
              alt="A smartphone, table and desktop computer"
```
We don't need to mention "picture" in the description as we already know it's a picture. Same applies to the other alt descriptions